resources/lib/helpers/jsonhelper.py

```diff
@@ -132,6 +132,34 @@ def get_value(self, *args, **kwargs):
 
         return data
 
+    #noinspection PyUnboundLocalVariable
```

Comment:
Perhaps we can also call the `get_from` from the `get_value` to prevent duplicate code?

Reply:
I have been trying to do exactly that. However, there does not seem to be a nice way to pass `*args` and `**kwargs` directly, because they become a list and a dict. I have not found a way to fix that without updating all code that is using `get_value`. Do you have a suggestion?

Reply:
I think you can just pass *args, and **kwargs like that? So call `JsonHelper.get_from(*args, **kwargs)`

Reply:
Thanks! I am not a Python developer and have not been googling carefully enough.